model/reviews.js

```diff
@@ -13,3 +13,23 @@ exports.selectReviewById = (review_id) => {
       return rows[0];
     });
 };
+
+exports.updateReviewById = (review_id, { inc_votes } = 0) => {
+  return db
+    .query(
+      `UPDATE reviews
+    SET votes = votes + $1
+    WHERE review_id = $2
+    RETURNING*`,
```

Comment:
Am I gonna kick off about missing whitespace... I might, I'd change it. You can decide whether it bothers you haha

Reply:
Thanks for the amazing feedback!! I will take note of everything and try implement it through out my code. Also I am a bit picky so I will have to use the whitespace ahah. Thanks, again Haz!!